src/libs/actions/BankAccounts.js

```diff
@@ -587,6 +571,7 @@ function validateBankAccount(bankAccountID, validateCode) {
         .then((response) => {
             if (response.jsonCode === 200) {
                 Growl.show('Bank Account successfully validated!', CONST.GROWL.SUCCESS, 5000);
+                Onyx.set(ONYXKEYS.REIMBURSEMENT_ACCOUNT_DRAFT, {});
```

Comment:
NAB, would it be more correct to set this to `null` and have it deleted? If we don't need the draft anymore then it's just taking up a key in localStorage with an empty `{}`.